jwplayer-appletv-web-app/js/application.js

```diff
@@ -104,3 +103,7 @@ function extend(obj) {
   }
   return obj;
 }
+
+function genId(len) {
```

Comment:
Maybe time for a utils module?  Not really sure `genId` belongs in `application.js`.  Alternately it can go in `TVOSAnalytics` and be exposed publicly.